std/net/curl.d

```diff
@@ -1025,32 +1024,14 @@ private auto _basicHTTP(T)(const(char)[] url, const(void)[] sendData, HTTP clien
             import std.conv : to;
             content.reserve(value.to!size_t);
         }
-        if (auto v = key in headers)
-        {
-            *v ~= ", ";
-            *v ~= value;
-        }
-        else
-            headers[key] = value.idup;
     };
     client.onReceiveStatusLine = (HTTP.StatusLine l) { statusLine = l; };
     client.perform();
     enforce!CurlException(statusLine.code / 100 == 2,
                             format("HTTP request returned status code %d (%s)",
                                    statusLine.code, statusLine.reason));
 
-    // Default charset defined in HTTP RFC
```

Comment:
> Love the red but ... was it basically useless work duplicated elsewhere?

The charset detection is already done in the other location that's touched by this PR: HTTP.onReceiveHeader. The function here calls that.